<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Test_Form</class>
 <widget class="QWidget" name="Test_Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>513</width>
    <height>315</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>70</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Вопрос</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Third_Choise">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>190</y>
     <width>311</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_24">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>70</y>
     <width>101</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Варианты ответа</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_25">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>51</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Первый</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_26">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>51</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Второй</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="Fourth_Correct">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>230</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_28">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>61</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Четвертый</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="quest_text">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="Fourth_Choise">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>230</y>
     <width>311</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="Two_Correct">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>150</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_27">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>51</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Третий</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_29">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>70</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Правильный ответ</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="One_Correct">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>110</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="Two_Choise">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>150</y>
     <width>311</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="Three_Correct">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>190</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="First_Choise">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>110</y>
     <width>311</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Save_Question">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>270</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
